### DISCUSSION TITLE
ErrorBoundary component

### DISCUSSION THREAD

**Discussion by vieraboschkova** [appbaseio/reactivesearch#2055] [Q&A]
I am having trouble importing the ErrorBoundary component. The message I get is:
"Attempted import error: 'ErrorBoundary' is not exported from '@appbaseio/reactivesearch'.  Other components like DataSearch or ToggleButton, are imported ok.
`import {
  ReactiveBase,
  DataSearch,
  ReactiveList,
  SelectedFilters,
  ToggleButton,
  ReactiveComponent,
  // ErrorBoundary,
} from '@appbaseio/reactivesearch';`

**Comment by SavvyShah** [comment]
Could you specify the version of reactivesearch library. `ErrorBoundary` component is available for `v3.36.1` and after. You can see a working [codesandbox example](https://codesandbox.io/s/github/appbaseio/reactivesearch/tree/next/packages/web/examples/ErrorBoundary?file=/src/index.js) for the same.

**Reply by vieraboschkova** [reply]
I checked again, I have installed version 3.36.1

**Reply by siddharthlatest** [reply]
@vieraboschkova Can you share a CSB of your usage?
